framework/src/main/java/io/github/kgress/scaffold/BaseWebElement.java

```diff
@@ -209,6 +212,21 @@ public BaseWebElement(By by, By parentBy) {
     setWebElementWait();
   }
 
+  /**
+   * Creates a new element with a parent element using the supplied {@link By} locators for both
+   * elements. Useful when you want a more verbose element definition in context of your websites'
+   * DOM. The element is used as a reference for use later. Once the element is set, we create a new
+   * {@link WebElementWait} for it.
+   *
+   * @param by       the {@link By} locator to be used by this element
+   * @param parentBy the {@link By} locator for the parent element
+   * @param isHidden a {@link boolean} to specify if this element could be hidden
+   */
+  public BaseWebElement(By by, By parentBy, boolean isHidden) {
+    this(by, parentBy);
+    this.isHidden = isHidden;
+  }
+
```

Comment:
We'll probably want overloaded versions of `BaseWebElement(String cssSelector)` as well as `BaseWebElement(By)`.

Reply:
I thought about that, I took the route of minimizing the number of constructors needed by ensuring i captured the arguments all in one.  to accomplish the same as the constructor you are proposing, we just need to pass (cssSelector, null, true/false) .....

Reply:
I agree. I don't think many folks are using the constructor with parentBy. Most common usage I've seen is `By by` and `String cssSelector`